container_pipeline/templates/rest_framework_docs/docs.html

```diff
@@ -0,0 +1,14 @@
+{% extends "rest_framework_docs/base.html" %}
+
+{% block title %}CentOS Container Pipeline Service{% endblock %}
+
+{% block logo %}<a class="navbar-brand" href="#">Container Pipeline Service API Documentation</a>{% endblock %}
+
+{% block jumbotron %}
+<div class="jumbotron">
+    <h1>Container Pipeline Service Web API</h1>
+    <h3>Documentation of the 'Container Pipeline Service' Web API.</h3>
+</div>
+{% endblock %}
+
+{% block footer %}<div class="footer">Copyright © 2017 Container Pipeline Service.</div>{% endblock %}
```

Comment:
this is static page? I dont see variables based on urls. we are doing it in next phase right?

Reply:
Should i remove the docs bit?